pkg/api/routes_studio.go

```diff
@@ -23,6 +26,21 @@ func (rs studioRoutes) Routes() chi.Router {
 
 func (rs studioRoutes) Image(w http.ResponseWriter, r *http.Request) {
 	studio := r.Context().Value(studioKey).(*models.Studio)
+	etag := fmt.Sprintf("%x", md5.Sum(studio.Image))
+	if match := r.Header.Get("If-None-Match"); match != "" {
+		if strings.Contains(match, etag) {
+			w.WriteHeader(http.StatusNotModified)
+			return
+		}
+	}
+
+	contentType := http.DetectContentType(studio.Image)
+	if contentType == "text/xml; charset=utf-8" {
```

Comment:
for some svgs i tested  `contentType == "text/xml; charset=utf-8" || contentType == "text/plain; charset=utf-8"` is needed

Reply:
Updated